USE_CASES.md

```diff
@@ -1416,13 +1418,189 @@ func main() {
  "subdomain": "SUBDOMAIN", 
  "username": "YOUR_SENDGRID_SUBUSER_NAME"
 }`)
-	response, err := sendgrid.API(request)
-	if err != nil {
-		log.Println(err)
-	} else {
-		fmt.Println(response.StatusCode)
-		fmt.Println(response.Body)
-		fmt.Println(response.Headers)
-	}
+  response, err := sendgrid.API(request)
+  if err != nil {
+    log.Println(err)
+  } else {
+    fmt.Println(response.StatusCode)
+    fmt.Println(response.Body)
+    fmt.Println(response.Headers)
+  }
 }
 ```
+
+<a name="deployments"></a>
+
+ # Deployments
+
+<a name="deployments"></a>
+
+ ## DigitalOcean
+
+ ### Setup
+
+ To begin, spin up a new droplet by going to the Droplets tab and selecting 'Create Droplet'. Select an OS (for this tutorial I'll be using Ubuntu 16.04.3 x64), price tier, and region, and optionally add your SSH key (outside the scope of this article, but [DigitalOcean has some great tutorials on the subject](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-freebsd-server)). Once the droplet is spun up, open the web console or SSH into it (if you didn't specify an SSH key, your password should be emailed to you), log in, and let's begin setting up our droplet for deployment!
+
+ To begin, on a new droplet it's best to always run a quick update and upgrade:
+
+ ```
+ $ apt update && apt upgrade -y
+ ```
+
+ Next, we'll install Go. I found the easiest way to download an updated version of Go is to use a custom repository, as shown below. 
```

Comment:
Instead of a custom repo, could we install go via the official documentation?  Maybe something along the lines of:
```
$ curl https://dl.google.com/go/go1.11.1.linux-amd64.tar.gz
$ tar -C /usr/local -xzf go1.11.1.linux-amd64.tar.gz
$ export PATH=$PATH:/usr/local/go/bin
$ source $HOME/.profile.
$ go version
go version go1.11.1 linux/amd64
```

The above example is taken from: https://golang.org/doc/install#install